public-api/src/main/resources/public-api.yaml

```diff
@@ -329,8 +329,9 @@ definitions:
           type: string
           description: match column type
         standardConcept:
```

Comment:
You might want to rename this "standardConcepts".

Reply:
Renamed the field to standardConcepts.